1/Dockerfile

```diff
@@ -14,12 +14,14 @@ MAINTAINER Ben Parees <bparees@redhat.com>
 
 ENV JENKINS_VERSION=1.609 \
     HOME=/var/lib/jenkins \
-    JENKINS_HOME=/var/lib/jenkins
+    JENKINS_HOME=/var/lib/jenkins \
+    STI_SCRIPTS_URL=image:///usr/libexec/s2i
```

Comment:
@bparees +1 but I would keep this in RHEL7 Dockerfile for backward compatibility with OSE 3.0.1